exec/sequence_test.go

```diff
@@ -24,7 +24,7 @@ func Test_Sequence(t *testing.T) {
 		}),
 		F("Test c", func() error {
 			result <- "c"
-			return errors.New("c has an error")
+			return fmt.Errorf("c has an error")
```

Comment:
IMO, at this point the old solution was better. fmt.ErroF is supposed to be used with placeholders.